assignments/pr_review_tutorial/chernyavskaya/README.md

```diff
@@ -1,13 +1,11 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+There are some two functions in simple_functions.py, "factorial" and "fibonacci".
```

Comment:
Maybe use double backticks for emphasis rather than quote marks, it will provide a nicer markdown rendering.